build/common.project.props

```diff
@@ -12,7 +12,8 @@
     <NETFXTargetFramework>net472</NETFXTargetFramework>
     <NETCoreTargetFramework>netcoreapp2.1</NETCoreTargetFramework>
     <NETCoreTargetFrameworks Condition=" '$(RequiresSigningXplatAPIs)' != 'true' ">netcoreapp2.1</NETCoreTargetFrameworks>
-    <NETCoreTargetFrameworks Condition=" '$(RequiresSigningXplatAPIs)' == 'true' ">netcoreapp2.1;netcoreapp3.0</NETCoreTargetFrameworks>
+    <NETCoreTargetFrameworks Condition=" '$(RequiresSigningXplatAPIs)' == 'true' and '$(TestForOneSDK)' != 'true' ">netcoreapp2.1;netcoreapp3.0</NETCoreTargetFrameworks>
```

Comment:
naming: Not sure what a better name is, but I don't think `TestForOneSDK` is appropriate. 
We really want to test for net core one framework, so maybe more like: 

TestForLatestNetCoreOnly or TestForNetCore30Only. 

Thoughts?

Reply:
Thanks. I think "TestForLatestNetCoreOnly" is better :) Changed accordingly.